src/admin.py

```diff
@@ -36,8 +36,6 @@
 from model import ConsolidatedMetaKGDoc, MetaKGDoc
 from utils import indices
 
-logging.basicConfig(level="INFO")
-
```

Comment:
@NikkiBytes although we said we should remove this line in another file, let's still keep this line in this `admin.py` file. These are admin tasks we need to run as developers, it's probably always a good idea to turn it on.

Reply:
Hey I added it [here](https://github.com/SmartAPI/smartAPI/blob/851ee5c16d16ab57af3eed361ed4e8257e4db0f5/src/admin.py#L39) to `admin.py`.